tutorials/marble/getting-started.md

```diff
@@ -0,0 +1,89 @@
+# Getting Started
+
+## Table of Contents
```

Comment:
Removed

Reply:
You've remove this but replaced it with a different manually written table of contents above it. 

Please read through the tutorial in the link above and change this so that jupyterbook creates a table of contents for this page for you.

If you are having trouble with the tutorial, let me know and we can chat about it